docs/platform_concepts/inbox/chats-configuration.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Instead of creating a new file, maybe you can push this content to `Inbox > Inbox Settings > Chat` as shown in the screenshot below
<img width="1434" alt="Screenshot 2022-06-23 at 6 18 28 PM" src="https://user-images.githubusercontent.com/18494493/175302689-c9c45572-3dff-453b-b70c-26b1a6e171cb.png">
Configurations

Reply:
As directed, the changes has been updated and raised a Pull request.
Please verify.

Reply:
A new doc - Data Explorer & Dashboards added to Insights.
Please verify.